ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:                                                                                
+  operator-registry:                                                                        
+    name: "4.3"                                                                             
+    namespace: ocp                                                                          
+    tag: operator-registry                                                                  
```

Comment:
What's this image for? 
Just curious, I assume is related to the commented code below, but I assumed that if a Dockerfile was provided the `from:` image did not make much sense since the Dockerfile was used to generate the image.

Reply:
we need it to build the operator's registry container, which is used to install the operator. The registry container has the operator's CSV and everything. We'll have to build a registry container with a CSV file that has all the RBAC+deployment logic related to installing our operator. The flow is, install the registry container, which installs the CSV, which installs the operator.

so we need this operator-registry base image because it contains the logic we need to build our registry container. There's an sqllite database initialization that occurs within the registry container's Docker build step, and the operator-registry base container has those initialization scripts the our build step consumes.

Bascially, it's not going to make sense right away, but it will soon when we start deploying our operator on a live cluster.

Reply:
Thanks for taking the time to explain this!